ui/permissions_test.py

```diff
@@ -533,6 +533,115 @@ def test_video_admin_permission_matching_lists_post(mock_moira_client,
     assert video_permission.has_object_permission(request_data.request, request_data.view, video) is True
 
 
+@pytest.mark.parametrize("enabled", [True, False])
```

Comment:
Can you globally change `enabled` to something like `video_permission_setting`? Take this line for example: `assert video_permission.has_object_permission(request_data_anon.request, request_data_anon.view, video) is enabled`. The words `is enabled` are misleading, I think. We're checking to see if two boolean values match. `assert video_permission.has_object_permission() is video_permission_setting` is a bit clearer about that than `assert video_permission.has_object_permission() is enabled`

Reply:
ok